tools/k8s-vagrant.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(dirname $0)
+
+function vagrant_up() {
+	pushd "$SCRIPT_DIR/../vagrant/"
+	vagrant up
+	popd
+}
+
+function wait_for_k8s() {
+	local cluster_status="$SCRIPT_DIR/../vagrant/scripts/cluster_status.sh"
+	echo -n 'Waiting for k8s'
+	while ! bash $cluster_status >/dev/null 2>&1; do
+		echo -n '.'
+		sleep 1
+	done
+	eval $(bash "$SCRIPT_DIR/../vagrant/scripts/cluster_use.sh")
+}
+
+function k8s_cluster_check() {
+	kubectl run nginx-test --image=nginx --port=80
+	local pod=$(kubectl get pods -l run=nginx-test -o jsonpath='{ .items[0].metadata.name }')
+	echo "Got pod $pod"
+	kubectl expose pod $pod --target-port=80 --name=nginx-test --type=LoadBalancer
+	local host=$(kubectl get nodes \
+		-o jsonpath='{ .items[0].status.addresses[?(@.type == "InternalIP")].address }')
+	local nodeport=$(kubectl get service nginx-test -o jsonpath='{ .spec.ports[0].nodePort }')
+	echo -n "Awaiting http://$host:$nodeport "
+	while ! wget -qO /dev/null --tries 1 --timeout 1 http://$host:$nodeport ; do
```

Comment:
I guess better to not use infinite loop. If it fails for example 30 times (1s sleep = 30 seconds) so we should terminate with failure report.